index.html

```diff
@@ -94,9 +94,7 @@
     Feel free to delete this message once you have read it.
     Good luck and Happy Coding!
 
-    - The freeCodeCamp Team
-
-    -->
+    - The freeCodeCamp Team-->
     <script  src="js/index.js"></script>
```

Comment:
- [OPTIONAL] Always state the `WHY` you ask the students to make some changes to their line or block  of code. With this, it would enable the students to learn very quickly and give them insight of why they should implement your requests.